src/libs/ReportActionsUtils.js

```diff
@@ -52,6 +52,11 @@ function getSortedReportActions(reportActions, shouldSortInDescendingOrder = fal
     }
     const invertedMultiplier = shouldSortInDescendingOrder ? -1 : 1;
     reportActions.sort((first, second) => {
+        // If only one action is a report created action, return the created action first.
```

Comment:
This is already being discussed here https://github.com/Expensify/App/issues/13310 is not a priority for now. So maybe we can just focus on greying out the LHN.

Reply:
I added a `reportActionID` to created report actions to fix a console error where the report action item in the list was missing a key (the reportActionID). As a result of that change the created messages would appear after the IOU message because they have the same `created` timestamp. Adding this condition fixes that issue and helps with the ordering so I would like to keep the change.

Reply:
NAB from my end. But I guess we will anyhow need to clean the sort method because reportActions would be random now, right?